src/containers/topbutton/Top.css

```diff
@@ -0,0 +1,21 @@
+#topButton {
+  visibility:hidden;
+  position: fixed;
+  bottom: 20px;
+  right: 25px;
+  opacity: 0.5;
+  z-index: 99;
+  border: none;
+  outline: none;
+  color: #0a4c61;
```

Comment:
I think we should not define colors in css because the colors will change with theme. May be you can remove just colors and keep everything as it is and implement rendering of colors and hover effect as we implemented in case of `Button` [here](https://github.com/ashutosh1919/masterPortfolio/blob/master/src/components/button/Button.js)